DigitalLearningSolutions.Web/Controllers/RegisterController.cs

```diff
@@ -160,10 +160,12 @@ public IActionResult Summary(SummaryViewModel model)
             }
 
             var baseUrl = ConfigHelper.GetAppConfig()["CurrentSystemBaseUrl"];
-            var userIP = Request.HttpContext.Connection.RemoteIpAddress.ToString();
+            var userIp = Request.Headers.ContainsKey("X-Forwarded-For") ?
+                Request.Headers["X-Forwarded-For"].ToString() :

```

Comment:
Re conversation we had before, did you find any documentation around how MVC handles several values for X-Forwarded-For? https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-Forwarded-For

Reply:
From what I can tell, there is no unique way that it is handled in MVC, which means that the X-Forwarded-For could be a comma separated list as specified in the docs you linked. In the cast that a comma separated list is retrieved, because it is all handled as a string where the client Ip is first, the code should still work as is.